templates/home.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <title>It works!</title>
+</head>
+
+<body>
+  <a href={{dashboard}} target="_blank">Dashboard</a>
```

Comment:
Maby some if with an error if the link is empty?

Reply:
Yup these links should be probably be wrapped with some `if` and rendered only when the variable is set.

Reply:
Right now, when dashboard and storefront urls are not set, it just stays at the home page.  So we should wrap this in if statement and show this elements only when variables are set?

Reply:
I think so. If the env is not set the link won't work anyway. Then we could even display some guide/message to tell the user how to start the service (but we don't have that on designs yet).